### PR TITLE
feat: PusherMainView Localization (SDKCF-5385)

### PR DESCRIPTION
**Pull request by malayli**

### CODE REVIEW

File: PusherMainView/PusherMainView/String+Localization.swift [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        guard let bundle = Bundle.sdkAssets else {
+            return self
+        }
+        return NSLocalizedString(self, bundle: bundle, comment: "")
+    }
+}
+
+private extension Bundle {
+    static func bundle(bundleIdSubstring: String) -> Bundle? {
+        return (allBundles + allFrameworks).first(where: { $0.bundleIdentifier?.contains(bundleIdSubstring) == true })
```
Comment (apparition47) [inline PusherMainView/PusherMainView/String+Localization.swift:14]: nit: don't need `return`

File: PusherMainView/PusherMainView/Localizable.strings [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+"please.enter.an app.bundle.id" = "Please enter an app bundle ID";
+"please.select.an.apns.method" = "Please select an APNS method";
+"please.select.an.apns.certificate" = "Please select an APNS Certificate";
+"please.enter.a.device.token" = "Please enter a Device Token";
+"choose.identity" = "Choose the identity to use for delivering notifications: \n(Issued by Apple in the Provisioning Portal)";
+
+// Error
+"error.json.file.is.incorrect" = "JSON file is incorrect";
+"error.p8.file.is.incorrect" = "p8 file is incorrect";
+"error.no.identity" = "There isn't identity.";
```
Comment (apparition47) [inline PusherMainView/PusherMainView/Localizable.strings:22]: I know this was here before but we should change it to something like "No identity set."

**Review by imaginaris — APPROVED** [review]

**Review by apparition47 — APPROVED** [review]

File: PusherMainView/PusherMainView/String+Localization.swift (hunk 1 above)
Comment (malayli) [reply PusherMainView/PusherMainView/String+Localization.swift]: Removed.

File: PusherMainView/PusherMainView/Localizable.strings (hunk 2 above)
Comment (malayli) [reply PusherMainView/PusherMainView/Localizable.strings]: Updated.

**Pull request merged**